### ISSUE TITLE
KeePassXC-Browser: Reload Shortcut

### ISSUE THREAD

**Issue by pablopaul** [keepassxreboot/keepassxc#1854]
Is there a shortcut to trigger "Reload" for the KeePassXC-Browser extension?

**Comment by varjolintu** [comment]
No. If you want it, make a request to the extension repository https://github.com/keepassxreboot/keepassxc-browser/issues
